site/pages/create-consequence-network/__snapshots__/create-consequence-network.test.tsx.snap

```diff
@@ -277,17 +277,21 @@ exports[`pages > CreateConsequenceNetwork > should render correctly with inputs
               <div
                 className=""
               >
-                <input
-                  aria-describedby=""
-                  className="govuk-input govuk-date-input__input govuk-input--width-4"
-                  defaultValue="yes"
-                  disabled={false}
-                  id="disruption-delay-input"
-                  name="disruptionDelay"
-                  onBlur={[Function]}
-                  placeholder=""
-                  type="text"
-                />
+                <div
```

Comment:
why is this div added?

Reply:
for css when the now button is show notice how on the time selector a flex and gap is added